data/json/recipes/recipe_medsandchemicals.json

```diff
@@ -1275,5 +1275,83 @@
     "autolearn": true,
     "qualities": [ { "id": "CHISEL", "level": 2 }, { "id": "HAMMER", "level": 2 } ],
     "components": [ [ [ "material_rhodonite", 1 ] ] ]
+  },
+  {
+    "type": "recipe",
+    "result": "chem_phenol",
+    "category": "CC_CHEM",
+    "subcategory": "CSC_CHEM_CHEMICALS",
+    "skill_used": "cooking",
+    "difficulty": 4,
+    "time": "50 m",
+    "batch_time_factors": [ 85, 3 ],
+    "book_learn": [ [ "adv_chemistry", 3 ], [ "textbook_chemistry", 3 ], [ "recipe_labchem", 3 ], [ "reference_cooking", 6 ] ],
+    "charges": 100,
+    "//": "hydrolysis of acetylsalicylic to salicylic acid followed by thermal decomposition to phenol",
+    "qualities": [ { "id": "CHEM", "level": 3 } ],
+    "tools": [ [ [ "surface_heat", 120, "LIST" ] ] ],
+    "components": [ [ [ "aspirin", 200 ] ], [ [ "chem_sulphuric_acid", 1 ] ] ]
+  },
+  {
+    "type": "recipe",
+    "result": "chem_phenol",
+    "id_suffix": "by_pyrolysis",
+    "category": "CC_CHEM",
+    "subcategory": "CSC_CHEM_CHEMICALS",
+    "skill_used": "cooking",
+    "difficulty": 7,
+    "time": "30 m",
+    "book_learn": [ [ "adv_chemistry", 5 ] ],
+    "charges": 100,
+    "//": "pyrolysis of coal, 600-800C, add/substitute coal chemistry books if they appear, other phenols and cresols can be byproducts",
```

Comment:
Lots of things here. First off, pyrolysis of coal is going to produce a huge variety of products. To obtain a range of materials you are interested in, you would need some very fine temperature control. 600-800 c is also far too hot, as <https://www.intechopen.com/books/pyrolysis/pyrolysis-of-low-rank-coal-from-research-to-practice> seems to indicate that your useful fraction here is obtained at ~350c in general.  As illustrated there in this image, this 'useful fraction' would still be very impure and require additional work. ![](https://api.intechopen.com/media/chapter/54296/media/F1.png)


Also 600-800 c is in the softening temperature range for soda-lime glass.



I'm not saying this can't be done, I'm saying you'd have way more byproducts than you indicate, some not useful to the survivor (or a faction even). A more controlled, dedicated pyrolysis and distillation setup is needed to really get this to be feasible.

Reply:
Temperature would be dependent on type of coal used for reaction, so yes, for lignites it would be significantly lower than for anthracites. Pyrolysis is done in a crucible not in glassware, and products are significantly cooled before reaching refinement part of the setup.